proxy.nomad.hcl

```diff
@@ -0,0 +1,88 @@
+job "nginx-proxy" {
+
+  group "nginx" {
+    network {
+      port "http" {
+        static = 80
+        to     = 8080
+      }
+    }
+
+    task "nginx" {
+      driver = "docker"
+
+      config {
+        image          = "nginx:mainline"
+        command        = "nginx"
+        args           = ["-c", "/local/nginx.conf"]
+        ports          = ["http"]
+        auth_soft_fail = true
+      }
+
+      identity {
+        env  = true
+      }
+
+      resources {
+        cpu    = 200
+        memory = 200
+      }
+
+      template {
+        destination = "local/nginx.conf"
+        data        = <<EOF
+daemon off;
+
+events {}
+
+http {
+  server {
+    listen 8080;
+
+    location /.well-known/jwks.json {
```

Comment:
Actually you brought my attention to the fact that I included the UI proxying configs from https://developer.hashicorp.com/nomad/tutorials/manage-clusters/reverse-proxy-ui without telling the user about it.

I'm going to remove everything but the `/.well-known/` locations because it's not safe to proxy the other paths without at least ACLs enabled.

Reply:
> I was hurt in a previous life, and now every time I see nginx config, it becomes a regex minigame.

If you really want to Nginx minigame :grinning: , I'm pretty sure these can both be `location =`. But I'd probably not bother.